internal/human/bytes.go

```diff
@@ -0,0 +1,14 @@
+package human
+
+import (
+	"fmt"
+	"math"
+)
+
+func Bytes(b int64) string {
+	sizes := []string{"B", "kB", "MB", "GB"}
```

Comment:
Note: I stopped at Gigabyte, which is already above our common use cases. So 1000 GB will still be 1000 GB and not 1 TB.